js/backup_restore.js

```diff
@@ -566,7 +566,10 @@ function configuration_restore(callback) {
                     mincheck:               1100,
                     maxcheck:               1900,
                     rx_min_usec:            885,
-                    rx_max_usec:            2115
+                    rx_max_usec:            2115,
```

Comment:
Not sure how much sense it makes to add this in here. This hasn't been updated in a while, so I don't know if this is of any use at all  for migrating configurations as it is now.